virtual_rainforest/models/animals/scaling_functions.py

```diff
@@ -34,11 +34,13 @@ def damuths_law(mass: float, terms: tuple) -> int:
     return ceil(terms[1] * mass ** terms[0])
 
 
-def metabolic_rate(
+def metabolic_rate_energy(
     mass: float, temperature: float, terms: tuple, metabolic_type: MetabolicType
 ) -> float:
     """Calculates the metabolic rate of animal cohorts.
 
+    Depreciated.
```

Comment:
Not sure if it will be worth to raise a `DeprecationWarning` (see https://docs.python.org/3.10/library/warnings.html#warning-categories) or if clarify in other way what you mean by `Deprecated`. Maybe appending a `FIXME` or `TODO`  here giving more context of what you mean will be enough.